src/handlers/quotes.js

```diff
@@ -76,15 +76,15 @@ module.exports = {
       }, EventSdk.AuditEventAction.start)
 
       // call the quote request handler in the model
-      model.handleQuoteRequest(request.headers, request.payload, span)
+      const result = await model.handleQuoteRequest(request.headers, request.payload, span)
```

Comment:
Major issues that I see here:

1. Any reason we can't split out the rules engine as a separate sync response?
2. Why are we not handling rule failures (I.e. business rules) by the expected FPSIOP Error Callback instead of handling this sync? <-- I think this will need to be addressed by the Design Authority.

@elnyry-sam-k, let me know your thoughts here.

Reply:
+1 @mdebarros - we shouldn't use `await` here, since we want to reply to the DFSP with a `202` as soon as possible

Reply:
We need to discuss this further as per discussion with Miguel